src/uarch-step.cpp

```diff
@@ -44,26 +44,26 @@ static inline uint64 readUint64(UarchState &a, uint64 paddr) {
 template <typename UarchState>
 static inline uint32 readUint32(UarchState &a, uint64 paddr) {
     require((paddr & 3) == 0, "misaligned readUint32 address");
-    uint64 palign = paddr & ~uint64(7);
-    uint32 bitoffset = uint32ShiftLeft(uint32(paddr) & uint32(7), 3);
-    uint64 val64 = readUint64(a, palign);
+    const uint64 palign = paddr & ~uint64(7);
```

Comment:
I think the `machine-solidity-step` generator might not be prepared for these `const` changes in code, we have to double check there if we doesn't break.

If this turns out to be an issue, maybe we could disable linting on this file.

Reply:
@mpernambuco Would this be a problem?